dotnet/src/Connectors/VectorData.Abstractions/VectorSearch/VectorSearchOptions.cs

```diff
@@ -30,6 +30,12 @@ public class VectorSearchOptions<TRecord>
     /// </value>
     public string? VectorPropertyName { get; init; }
 
+    /// <summary>
```

Comment:
@adamsitnik did I see correctly that you removed the [Obsolete] from VectorPropertyName just above? Don't we want to obsolete it?

Reply:
@roji great catch! It must have been removed during the merge (xml docs were updated), I am going to add it back of course.